File: 100-lambda.rb
# This is what we want to implement........

fact = ->(x) {
  if x == 1         # if, true, false
    1               # numbers
  else              # else
    x * fact.(x-1)  # decrement, multiply, recursion
  end
}

puts fact.(8)

# let's start with numbers.....

File: js/100-lambda.js
// This is what we want to implement........

const fact = (x) => {
  if (x == 1) {      // if, true, false
    return 1              // numbers
  } else {           // else
    return x * fact(x-1)  // decrement, multiply, recursion
  }
}

console.log( fact(8) )

// let's start with numbers.....
